src/app/app.routing.ts

```diff
@@ -35,7 +37,8 @@ const appRoutes: Routes = [
   { path: 'about', component: IntroComponent },
   { path: 'register-sellers', component: RegisterSellerComponent},
   { path: 'cart', component: BasketProductComponent},
-  { path: 'search', component: SearchAdvertisementComponent},
+  { path: 'searchAdvertisements', component: SearchAdvertisementComponent},
```

Comment:
Probably a merging mistake. `searchAdvertisements` route should be removed, since its component is already integrated in the `navbar`.